content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,73 @@
+weight: 50
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+Sidecar containers are the secondary containers that runs along with the main
+application container within the same {{< glossary_tooltip text="Pod" term_id="pod" >}}.
+These containers are used to enhance or extend the functionality of the primary container
+by providing additional services, or functionality such as logging, monitoring, security,
+or data synchronization, without directly altering the primary application code.
+
+<!-- body -->
+
+Starting with Kubernetes 1.28 in alpha, a feature gate named `SidecarContainers`
+allows you to specify a `restartPolicy` for containers listed with a Pod's `initContainers`
+field. These restartable _sidecar_ containers are independent of other
+[init containers](/docs/concepts/workloads/pods/init-containers/) and main application
+container within the same pod. These can be started, stopped, or restarted independently
+of the main container and other init containers.
+Container [probes](/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe)
+can also be added to control their lifecycle.
+
+If an init container is created with its `restartPolicy` set to `Always`, it will
+start and remain running during the entire life of the Pod, which is useful for
+running supporting services separated from the main application containers.
+
+If a `readinessProbe` is specified for this init container, its result will be used
+to determine the `ready` state of the Pod.
+
+Since these containers are defined as init containers, they benefit from the same
+ordering and sequential guarantees as other init containers, allowing them to
+be mixed with other init containers into complex Pod initialization flows.
+
+Compared to regular init containers, sidecar-style init containers continue to
+run and the next init container can begin starting once the kubelet has set
+the `started` container status for the sidecar-style init container to true.
+That status either becomes true because there is a process running in the
+container and no startup probe defined, or
+as a result of its `startupProbe` succeeding.
+
+This feature can be used to implement the sidecar container pattern in a more
+robust way, as the kubelet always restarts a sidecar container if it fails.
+
+Here's an example of a Deployment with two containers, one of which is a sidecar:
+
+{{% code_sample language="yaml" file="application/deployment-sidecar.yaml" %}}
+
+This feature is also useful for running Jobs with sidecars, as the sidecar
+container will not prevent the Job from completing after the main container
+has finished.
+
+Here's an example of a Job with two containers, one of which is a sidecar:
+
+{{% code_sample language="yaml" file="application/job/job-sidecar.yaml" %}}
+By default, this feature is not available in Kubernetes. To avail this feature,
+we need to enable the feature gate `SidecarContainers` by setting the value **true**
```

Comment:
We always avoid "we".
You may want to reread the contribute guidelines about the content styles.

Reply:
Updated, Thanks.